src/client/lib/_mapbox/map-factory.js

```diff
@@ -0,0 +1,51 @@
+import BaseLayerControl from './baselayer-control'
```

Comment:
I find this a complicated file, though I don't have any suggestions of how to make this simpler :-/